### PR TITLE
updated android runtime config docs

### PR DESCRIPTION
**Pull request by radixdev**
# Pull Request/Issue Resolution

**Description of Change:**
AppboyConfig is now BrazeConfig

### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Insert Feature Release Date Here__)
- [x] No

> If yes, please note the date of the feature release.

---
---

<details>
<summary>PR Checklist</summary>

- [ ] Check that all links work!
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is _done and ready to be reviewed for merge_.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/android/push_notifications/integration/standard_integration.md [hunk 1]
```diff
@@ -700,8 +700,8 @@ Braze push data keys are documented [here](https://appboy.github.io/appboy-andro
 [68]: {{site.baseurl}}/developer_guide/platform_integration_guides/android/advanced_use_cases/runtime_configuration/#runtime-configuration
 [70]: https://github.com/Appboy/appboy-android-sdk/blob/master/samples/firebase-push/src/main/AndroidManifest.xml "AndroidManifest.xml"
 [71]: https://github.com/Appboy/appboy-android-sdk/blob/master/samples/custom-broadcast/src/main/AndroidManifest.xml "AndroidManifest.xml"
-[72]: https://appboy.github.io/appboy-android-sdk/javadocs/com/appboy/configuration/AppboyConfig.Builder.html#setDefaultNotificationChannelName-java.lang.String-
-[73]: https://appboy.github.io/appboy-android-sdk/javadocs/com/appboy/configuration/AppboyConfig.Builder.html#setDefaultNotificationChannelDescription-java.lang.String-
+[72]: https://appboy.github.io/appboy-android-sdk/javadocs/com/appboy/configuration/BrazeConfig.Builder.html#setDefaultNotificationChannelName-java.lang.String-
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/push_notifications/integration/standard_integration.md:703]: update link

File: _docs/_developer_guide/platform_integration_guides/fireos/advanced_use_cases/runtime_configuration.md [hunk 2]
```diff
@@ -14,42 +14,42 @@ Runtime configuration is an optional way to configure your app at runtime in pla
...
 
 {% endtab %}
 {% endtabs %}
 
 Another example can be found in our [Hello Appboy sample app][3].
 
-[1]: https://appboy.github.io/appboy-android-sdk/javadocs/com/appboy/Appboy.html#configure-android.content.Context-com.appboy.configuration.AppboyConfig-
-[2]: https://appboy.github.io/appboy-android-sdk/javadocs/com/appboy/configuration/AppboyConfig.Builder.html
+[1]: https://appboy.github.io/appboy-android-sdk/javadocs/com/appboy/Appboy.html#configure-android.content.Context-com.appboy.configuration.BrazeConfig-
+[2]: https://appboy.github.io/appboy-android-sdk/javadocs/com/appboy/configuration/BrazeConfig.Builder.html
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/fireos/advanced_use_cases/runtime_configuration.md:53]: update link

File: _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/runtime_configuration.md [hunk 3]
```diff
@@ -12,41 +12,41 @@ Runtime configuration is an optional way to configure your app at runtime in pla
...
 ```
 
 {% endtab %}
 {% endtabs %}
 
 Another example can be found in our [Hello Appboy sample app][3].
 
-[1]: https://appboy.github.io/appboy-android-sdk/javadocs/com/appboy/Appboy.html#configure-android.content.Context-com.appboy.configuration.AppboyConfig-
-[2]: https://appboy.github.io/appboy-android-sdk/javadocs/com/appboy/configuration/AppboyConfig.Builder.html
+[1]: https://appboy.github.io/appboy-android-sdk/javadocs/com/appboy/Appboy.html#configure-android.content.Context-com.appboy.configuration.BrazeConfig-
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/runtime_configuration.md:50]: update link

**Review by Bucimis — APPROVED** [review]
love it

**Pull request merged**